requirements.lock

```diff
@@ -0,0 +1,389 @@
+# generated by rye
```

Comment:
is there a way to check if all the pinned version here match the poetry one? To minimize issues coming from dependencies.

Reply:
We're checking the locked version deps from poetry to rye to make sure they are the same for our deps and the transitive deps also. We'll run this in staging for a while  and if looks ok for a couple days we'll run in a canary in prod also before rolling it out.